docs/source/services/timelock_service/migration.rst

```diff
@@ -0,0 +1,110 @@
+.. _timelock-migration:
+
+Migration to External Timelock Services
+=======================================
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team before attempting a migration.
+
+Why Migration?
+--------------
+
+AtlasDB assumes that timestamps returned by the timestamp service are monotonically increasing. In order to preserve
+this guarantee when moving from an embedded timestamp service to an external timestamp service, we need to ensure
+that any timestamps the external timestamp service issues must be larger than any timestamps that the embedded
```

Comment:
This is a bit of tongue twister! How about "we need to ensure that timestamps issued by the external timestamp service are larger than those issued by the internal one"?